components/infobox/wikis/leagueoflegends/infobox_person_player_custom.lua

```diff
@@ -90,7 +90,10 @@ end
 function CustomInjector:parse(id, widgets)
 	if id == 'status' then
 		local status = _args.status
-		status = status:gsub("^%l", string.upper)
+		if String.isNotEmpty(status) then
+			status = mw.language:ucfirst(status)
```

Comment:
This would not be the correct way to call ucfirst. Note the `:`, you need an instance of a class on the left side.

Reply:
`mw.getContentLanguage():ucfirst(string)`